client/me/security-2fa-enable/index.jsx

```diff
@@ -457,5 +455,10 @@ export default localize(
 				</div>
 			);
 		},
+
+		handleChange( e ) {
+			const { name, value } = e.currentTarget;
```

Comment:
i like this, I'd never seen it before for reusing an onChange handler

Reply:
It's used at a lot of other places, for example [here](https://github.com/Automattic/wp-calypso/blob/38040768fba5ee8539ebd69204a817f644aa15f3/client/my-sites/checkout/checkout/paypal-payment-box.jsx#L52-L57). The principle is: be a know-nothing method and extract everything you need from the target element 🙂